BinaryEncoding.md

```diff
@@ -789,9 +789,14 @@ for [future :unicorn:][future multiple tables] use and must be 0 in the MVP.
 | ---- | ---- | ---- | ---- |
 | `is_lock_free` | `0xff00` | | |
 | `i32.wait` | `0xff01` | `memory immediate` | |
-| `i64.wait` | `0xff02` | `memory immediate` | |
-| `i32.wake` | `0xff03` | `memory immediate` | |
-| `i64.wake` | `0xff04` | `memory immediate` | |
+| `i32.wake` | `0xff02` | `memory immediate` | |
+
+The `memory_immediate` type is encoded as follows:
+
+| Name | Type | Description |
+| ---- | ---- | ---- |
+| flags | `varuint32` | a bitfield which currently must be 0 |
+| offset | `varuint32` | the value of the offset |
```

Comment:
I figure it makes sense for consistency. And is convenient for the same reason that it is for other memory accesses, so you can have a pointer to a struct and use that for wait/wake instead of having to add it in yourself.